test/e2e/Tests/build-e2e-test.ps1

```diff
@@ -192,6 +195,26 @@ function StartMSSQLContainer($mssqlPwd) {
   docker ps
 }
 
+function StartDTSContainer() {
+  Write-Host "Pulling down the durabletaskspublic.azurecr.io/dts-emulator:latest-amd64 image..."
+  docker pull durabletaskspublic.azurecr.io/dts-emulator:latest-amd64
+
+  # Start the DTS Server docker container with the specified edition
+  Write-Host "Starting DTS docker container on port 8080" -ForegroundColor DarkYellow
+  docker run -itP -p 8080:8080 -p 8082:8082 -e ClientAuth__DisableAuthentication=true -d durabletaskspublic.azurecr.io/dts-emulator:latest-amd64
```

Comment:
We have an official MCR distribution of this container now, and it fixes some bugs, like the need for the extra environment variable to disable auth.
```suggestion
  docker run -itP -p 8080:8080 -p 8082:8082 -d mcr.microsoft.com/dts/dts-emulator
```